web/src/utils/index.ts

```diff
@@ -0,0 +1,5 @@
+export { formatBalance } from './formatBalance'
```

Comment:
is this import required?

Reply:
it's re-exporting, just so that the import is made on `@utils` rather than `@utils/formatBalance`. But it's a nit.